### PR TITLE
SEO optimization: KW "kubernetes deployment"

### PR DESCRIPTION
**Pull request by 97thFloor**
KW: kubernetes deployment
Secondary KWs: deployment, kubernetes rollout, kubernetes deployments

### Proposed changes
Increased instances of primary and secondary keywords, as well as title tag update.

@zchase @cnunciato

### CODE REVIEW

File: content/docs/guides/crosswalk/kubernetes/apps.md [hunk 1]
```diff
@@ -1017,7 +1017,7 @@ Coming soon.
 
 ## Learn More
 
-To learn more about how to work with Kubernetes and Pulumi, check out the
+To learn more about Kubernetes deployment and how to work with Kubernetes and Pulumi, check out the
```
Comment (zchase) [inline content/docs/guides/crosswalk/kubernetes/apps.md:1020]:
```suggestion
To learn more about how to work with Kubernetes and Pulumi, check out the
```

File: content/docs/guides/crosswalk/kubernetes/apps.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ menu:
     weight: 9
 ---
 
-The following are examples of how to create and use various types of Kubernetes
+The following are examples of Kubernetes deployment with Pulumi — how to create and use various types of Kubernetes
```
Comment (zchase) [inline content/docs/guides/crosswalk/kubernetes/apps.md:11]:
```suggestion
The following are examples of how to create and use various types of Kubernetes
```

File: content/docs/guides/crosswalk/kubernetes/apps.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Deploy Kubernetes Apps
+title: Kubernetes App Deployment
```
Comment (zchase) [inline content/docs/guides/crosswalk/kubernetes/apps.md:2]: @cnunciato WDYT of this? I'm not sure we should change the title.

Comment (cnunciato) [reply content/docs/guides/crosswalk/kubernetes/apps.md:2]: @zchase Agreed, good catch -- we should not change the title here. The rest are verbs in the nav and it should stay that way IMO.

image

Comment (zchase) [reply content/docs/guides/crosswalk/kubernetes/apps.md:2]:
```suggestion
title: Deploy Kubernetes Apps
```

**Pull request merged**